grouper/api/handlers.py

```diff
@@ -147,6 +155,34 @@ def get(self, name=None):
             })
 
 
+class BulkUsers(GraphHandler):
+    """API endpoint for bulk retrieval of user data.
+
+    This returns the same information as the Users and ServiceAccounts endpoints, but supports
+    multiple returning the data of multiple users to save on API call overhead.
+    """
+    def get(self):
+        names = self.get_argument("names", None)
+        if names is None:
+            usernames = self.graph.user_metadata.iterkeys()
+        else:
+            usernames = names.split(",")
+
+        cutoff = int(self.get_argument("cutoff", 100))
+        include_service_accounts = self.get_argument("include_role_users", "no") == "yes"
```

Comment:
If you have the names of the accounts, I think you should get the information regardless of whether it was a service account or not.  If you then want to filter out service accounts, the client can easily do that by looking at the `service_account` key of the user data.  So I would drop this argument entirely and just always include service accounts in the result, similar to what `/users/<username>` currently does.